pokemontcgsdk/card.py

```diff
@@ -32,14 +32,15 @@ class Card():
     regulationMark: Optional[str]
     name: str
     nationalPokedexNumbers: Optional[List[int]]
-    number: Optional[str]
+    number: str
     rarity: Optional[str]
+    regulationMark: Optional[str]
```

Comment:
I don't know why this is showing up in the diff..... this should already be here?